backend/src/api/email/email.handlers.ts

```diff
@@ -1,19 +1,51 @@
 import { Request, Response } from "express-serve-static-core";
-import { PostmarkInboundEmailBody } from "./email.types";
+import { MailgunInboundEmailBody } from "./email.types";
+import {
+  parseUnverifiedVenmoEmail,
+  updateOrderPaymentStatus,
+} from "./email.services";
+import { isValidVenmoEmail } from "../../utils/email";
+import { z } from "zod";
 
-// TODO
 export const parseEmailHandler = async (
-  req: Request<{}, any, PostmarkInboundEmailBody, {}>,
+  req: Request<{}, any, z.infer<typeof MailgunInboundEmailBody>, {}>,
```

Comment:
No need for `z.infer<typeof MailgunInboundEmailBody>`, just use `MailgunInboundEmailBody`